diff -Nur micq-0.4.2-orig/util.c micq-0.4.2/util.c
--- micq-0.4.2-orig/util.c	Mon Sep 13 03:42:33 1999
+++ micq-0.4.2/util.c	Wed Dec  1 15:24:19 1999
@@ -742,7 +742,7 @@
    va_list args;
    int k;
    char buf[2048]; /* this should big enough */
-   char    buffer[256];
+   char *buffer;
    time_t  timeval;
    char *path;
    char *home;
@@ -766,16 +766,19 @@
 
 #ifdef UNIX
    home = getenv( "HOME" );
-   path = malloc( strlen( home ) + 2 );
+   path = malloc( strlen( home ) + 8 ) ;
    strcpy( path, home );
    if ( path[ strlen( path ) - 1 ] != '/' )
-      strcat( path, "/" );
+	strcat( path, "/" );
+   strcat( path, "data/" );
+   if ( mkdir(path, 0700) && errno != EEXIST )
+	return -1;
 #endif
 
 #ifdef __amigaos__
    path = "PROGDIR:";
 #endif
-
+   buffer=malloc(strlen(path)+30);
    strcpy( buffer, path );
    strcat( buffer, "micq.log" );
    if ( -1 == stat( buffer, &statbuf ) ) {
